code/_globalvars/lists/poll_ignore.dm

```diff
@@ -0,0 +1,4 @@
+//Each lists stores ckeys for "Never for this round" option category
+
+var/list/sentience_potion_ignore = list() //sentience potion query
```

Comment:
Plus then you can use the Assoc keys as category names in some form of debug view:
```
for(var/category in role_ignores)
    var/list/L = role_ignores[category]
    if(L && L.len)
        sometext += "[category]:<br>"
        for(var/ignorer in L)
            sometext += "[ignorner]<br>"
```